index.bs

```diff
@@ -5783,10 +5779,12 @@ The {{MLOperand}} objects are created by the methods of {{MLGraphBuilder}}, inte
 
 <details open algorithm>
   <summary>
-    To <dfn>check dimensions</dfn> given [=/list=] |dimensions| and {{MLOperandDataType}} |type|, run the following steps:
+    To <dfn>check dimensions</dfn> given {{MLOperandDescriptor}} |descriptor|, run the following steps:
   </summary>
   <div class=algorithm-steps>
-    1. If |dimensions|'s [=list/size=] is 0, return false.
+    1. Let |dimensions| be |descriptor|.{{MLOperandDescriptor/dimensions}}.
+    1. Let |type| be |descriptor|.{{MLOperandDescriptor/dataType}}.
```

Comment:
Nit: would |dataType| be more explicit?

Reply:
Yeah I'd prefer dataType too, was just using type as it was already there for one of the lines. I'll update it.